### ISSUE TITLE
Training with MultiScale loss

### ISSUE THREAD

**Issue by zinuok** [NVIDIA/flownet2-pytorch#276]
Hello,
I'm trying to fine-tune the whole FlowNet2 network from the provided pre-trained weights, 
with the multi-scale loss by:
```bash
...
--loss=MultiScale \
--loss_norm=L1 \
...
```

However, the model returns only single flow output from the last fusion layer
```bash
        return flownetfusion_flow
```

and in the MultiScale class, the type of the flow output, which is passed as an argument, is checked
```bash
        if type(output) is tuple:
            target = self.div_flow * target
            for i, output_ in enumerate(output):
                target_ = self.multiScales[i](target)
                epevalue += self.loss_weights[i]*EPE(output_, target_)
                lossvalue += self.loss_weights[i]*self.loss(output_, target_)
            return [lossvalue, epevalue]
        else:
            epevalue += EPE(output, target)
            lossvalue += self.loss(output, target)
            return  [lossvalue, epevalue]
```

The part that I have the most questions about is that 
the type of model output is not always a tuple, so it falls to the else branch, which is the same as a simple L1loss. 
Do I need to change the return value of the model to use MultiScale loss? 

I've already seen the related issue: [issue 28](https://github.com/NVIDIA/flownet2-pytorch/issues/28), but I couldn't find the propoer answers.

Thank you
